rancher-config/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     labels:
       traefik.enable: true
       traefik.frontend.rule: $TRAEFIK_RULE
+      traefik.backend: ${RANCHER_STACK_NAME}-nearby-services-api
       traefik.port: 3001
       io.rancher.container.pull_image: always
       io.rancher.scheduler.affinity:container_label_soft_ne: "io.rancher.stack_service.name=$${stack_name}/$${service_name}"
```

Comment:
If this label isn't in use can it be removed?

Reply:
This tries to spread the containers across different hosts if possible, so should stay.